### PR TITLE
UIDATIMP-31: Add reusable Job component.

### PR DESCRIPTION
**Pull request by VictorSoroka1**

### CODE REVIEW

File: src/components/Jobs/Jobs.js [hunk 1]
```diff
@@ -1,26 +1,51 @@
...
-        />
-      </AccordionSet>
+      <div className={css.jobsPane}>
+        <AccordionSet>
+          <Accordion
+            label={formatMessage({ id: 'ui-data-import.previewJobs' })}
+            separator={false}
+          >
+            <div className={css.jobList}>
+              <Job job={jobsMocks[0]} checkDateIsToday={this.checkDateIsToday} />
```
Comment (maximdidenkoepam) [inline src/components/Jobs/Jobs.js]: Please move each property to a new line. The same issue is in line 43

Comment (VictorSoroka1) [reply src/components/Jobs/Jobs.js]: As I understand you consider best practice to move properties on a new line if it more that one property, right?

File: src/components/Jobs/components/Job/Job.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+import Progress from '../Progress';
+import jobMetaTypes from './jobMetaTypes';
+
+import css from './Job.css';
+
+class Job extends React.Component {
+  static defaultProps = {
+    handlePreview: () => {
+      // TODO: to be implemented in further stories
```
Comment (maximdidenkoepam) [inline src/components/Jobs/components/Job/Job.js]: Can we use _.noop for this case?

Comment (VictorSoroka1) [reply src/components/Jobs/components/Job/Job.js]: Nice idea. Will do.

File: src/components/Jobs/components/Job/Job.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    const {
+      jobProfileName,
+      fileName,
+      status,
+      jobExecutionHRID,
+      runBy: { firstName, lastName },
+      progress: { current, total },
+    } = job;
+    const jobMeta = jobMetaTypes[status](job);
+    const dateLabel = `ui-data-import.${jobMeta.dateLabel}Running`;
```
Comment (maximdidenkoepam) [inline src/components/Jobs/components/Job/Job.js]: As I understand it is not a label. It is a key or translation id. Maybe we should rename it?

Comment (VictorSoroka1) [reply src/components/Jobs/components/Job/Job.js]: Do you mean to remove variable and put it like that? If so, I agree.

Comment (maximdidenkoepam) [reply src/components/Jobs/components/Job/Job.js]: I am ok with variable, but it is not a label, it is label id or key. I suggest to rename it for example to dateLabelId?

Comment (VictorSoroka1) [reply src/components/Jobs/components/Job/Job.js]: Agree, will rename.

File: src/components/Jobs/components/Job/Job.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  overflow-wrap: break-word;
+
+  & > span:not(:last-child):after {
+    content: " \B7 ";
+  }
+}
+
+.job {
+  padding: .5rem;
+  border: 1px solid var(--color-border);
```
Comment (oleksii-popov) [inline src/components/Jobs/components/Job/Job.css]: Good usage of variables

File: src/components/Jobs/Jobs.js [hunk 5]
```diff
@@ -1,26 +1,86 @@
...
 import { stripesShape } from '@folio/stripes/core';
-import { AccordionSet, Accordion } from '@folio/stripes/components';
+import {
+  AccordionSet,
+  Accordion,
+} from '@folio/stripes/components';
+
+import Job from './components/Job';
+import EndOfList from './components/EndOfList';
+import jobsMocks from './jobsMocks'; // TODO: to be replaced with real data in further stories
```
[the remark this answers is not in the document]
Comment (VictorSoroka1) [reply src/components/Jobs/Jobs.js]: Nice idea. Will do.

**Pull request merged**